perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -45,11 +45,21 @@
 from perfkitbenchmarker import vm_util
 from perfkitbenchmarker.sample import PercentileCalculator  # noqa
 
+
 flags.DEFINE_enum('storage', benchmark_spec_class.GCP,
                   [benchmark_spec_class.GCP, benchmark_spec_class.AWS,
                    benchmark_spec_class.AZURE],
                   'storage provider (GCP/AZURE/AWS) to use.')
 
+flags.DEFINE_string('object_storage_gcp_region', 'us-central1',
```

Comment:
I just talked with Stanley. Making object stores act like other resources is separate from the rest of this change, so I filed #621 to discuss. I'd like to submit this PR without that and maybe do that as a separate change.